docs/layouts/_default/reference-release.html

```diff
@@ -40,7 +40,11 @@
             {{ if isset .Params "heading" }}<h1>{{ .Params.heading }}</h1>{{else}}<h1>{{ .Title }}</h1>{{ end }}
           </header>
           <p><span class="usa-tag">Release Version</span> {{ if eq .Params.oscal.version "develop" }}Latest Development Snapshot{{ else }}OSCAL v{{ .Params.oscal.version }}{{ end }}</p>
+          {{ if isset .Params.oscal "remote" }}
+          <p><span class="usa-tag">Github</span> <a href="https://github.com/{{ .Params.oscal.remote }}">{{ .Params.oscal.remote }}</a> <span class="usa-tag">{{ if eq .Params.oscal.type "tag" }}Tag{{ else }}Branch{{end}}</span> <a href="https://github.com/{{ .Params.oscal.remote }}/tree/{{ .Params.oscal.branch }}">{{ .Params.oscal.branch }}</a></p>
+          {{ else }}
```

Comment:
@david-waltermire-nist I was joking about time warping with tagged release versions in the reference model docs missing the name of the repo (since they were generated before that `.Param.oscal.remote` argument existed) and it seems this was the workaround needed.